### DISCUSSION TITLE
supabase self-hosted monitoring

### DISCUSSION THREAD

**Discussion by therickys93** [supabase/supabase#27428] [Questions]
Hi I was wonder if there is a way to monitor my supabase self-hosted instance with uptime-kuma with all the services like it is shown in the official supabase status https://status.supabase.com

Thanks

**Comment by raman04-byte** [comment]
> Hi I was wonder if there is a way to monitor my supabase self-hosted instance with uptime-kuma with all the services like it is shown in the official supabase status https://status.supabase.com
> 
> Thanks

Did you find any possible solution to this as I am also trying to do this

**Reply by therickys93** [reply]
No, I do not have any update on this topic. I will share below my configuration:
- TCP Port 4000 for Supabase Analytics
- GET :8000/auth/v1/health for Supabase Auth (passing "apiKey" and "Authorization" with the same anonkey value as header)
- TCP Port 8000 for Supabase Kong
- TCP Post 5432 for Supabase Postgres
- GET :8000/rest/v1 for Supabase Rest API (passing "apiKey" and "Authorization" with the same anonkey value as header)
